pkg/reconciler/emit_metrics_test.go

```diff
@@ -2,12 +2,16 @@ package reconciler
 
 import (
 	"testing"
+	"time"
 
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/apis/pipelinesascode/keys"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/metrics"
 	tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
 	"gotest.tools/v3/assert"
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
 func TestEmitMetrics(t *testing.T) {
```

Comment:
you are not adding any new tests here, just fixing the current one?

Reply:
yeah, adjusted existing one to accommodate new metric.

Reply:
can we have a new test?

Reply:
I do not know how, without having to spend time on the code, what about a e2e test then?